api/openapi.yaml

```
@@ -4,20 +4,13 @@ info:
   version: "1.0.0"
   description: Authoritative contract for POST /api/plan. Non-auth, privacy-first. Spectral lint must pass with 0 errors; warnings allowed.
   contact: { name: PayPlan API, url: "https://github.com/mmtuentertainment/PayPlan" }
-tags:
-  - name: plan
-    description: Payment planning operations
-servers:
-  - url: https://api.example.com
-    description: Informational placeholder only; clients typically call relative path.
+tags: [{ name: plan, description: Payment planning operations }]
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)